01-python/leisure/museum.py

```diff
@@ -1,3 +1,31 @@
-def draw_museum():
-    print("museum not found")
+
+
+
+def draw_museum(frameWith=10, frameHeight=10):
+    artists=["Rodin", "Van Gogh", "Pollack", "Miró "]
```

Comment:
better to capitalize constants
```suggestion
    ARTISTS=["Rodin", "Van Gogh", "Pollack", "Miró "]
```